tests/buildenv.sh

```diff
@@ -54,7 +54,7 @@ check_emulation_support()
         return
     fi
 
-    if [ ! -x "${ARM_EMU_BIN}" ]; then
+    if [ ! -x "${ARM_EMU_BIN}" ] || [ "${ARM_EMU_BIN#/}" != "${ARM_EMU_BIN}" ]; then
```

Comment:
Again it is very confusing what actually is going on here, what are your actually testing